00_сompilation_process/00_01_compilation/00_01_00_compilation_error/00_01_00_compilation_error.cpp

```diff
@@ -0,0 +1,10 @@
+// Show compilation error than swap 3 and 4 lines
```

Comment:
Maybe it would be better do not use line numbers as a comment? + than -> then
```suggestion
// Show compilation error and swap inclusion and global variable definition
```